### PR TITLE
Master window is rendered with qml. Need optimization.

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

File: apps/DisplayCluster/DisplayGroupView.cpp [hunk 1]
```diff
@@ -69,7 +69,8 @@ DisplayGroupView::DisplayGroupView( OptionsPtr options )
 
 DisplayGroupView::~DisplayGroupView() {}
 
-void DisplayGroupView::setDataModel( DisplayGroupPtr displayGroup )
+void DisplayGroupView::setDataModel( DisplayGroupPtr displayGroup,
+                                     QSize numberOfTiles, int mullionWidth )
```
Comment (tribal-tec) [inline apps/DisplayCluster/DisplayGroupView.cpp]: const QSize&, const int

File: apps/DisplayCluster/resources/MasterDisplayGroup.qml [hunk 2]
```diff
@@ -25,11 +64,51 @@ DisplayGroup {
...
+        spacing: mullionWidth
+        Repeater {
+            layer.mipmap: true
+            id: screens
+            model: numberOfTilesX * numberOfTilesY
+            delegate: Rectangle {
+                //Generalized checker pattern computation
+                color: ((index%2 == 0) ^
+                       (!(Math.floor(index / numberOfTilesX)%2 == 0)
+                       && (numberOfTilesX%2 == 0))) ? "#B2C7CF" : "#97BFCC"
```
Comment (tribal-tec) [inline apps/DisplayCluster/resources/MasterDisplayGroup.qml]: add colors to style.js

**Comment by tribal-tec** [comment]
+1 otherwise

**Comment by chevtche** [comment]
Updated.

**Pull request merged**